python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -644,6 +645,15 @@ def test_schema(self):
     def test_to(self):
         # SPARK-41464: test DataFrame.to()
 
+        cdf = self.connect.read.table(self.tbl_name)
```

Comment:
what about simply:
```
cdf = ...
sdf = ...
for schema in [ schema1, schema2, schema3 ]:
            cdf_to = cdf.to(schema)
            sdf_to = sdf.to(schema)
            self.assertEqual(cdf_to.schema, sdf_to.schema)
            self.assert_eq(cdf_to.toPandas(), sdf_to.toPandas())
```

I prefer not adding extra comprasion function

Reply:
`cdf` and `sdf` changed.

Reply:
ok